src/view/com/posts/FeedItem.tsx

```diff
@@ -251,13 +252,10 @@ let FeedItemInner = ({
                 )}`,
               )}
               onBeforePress={onOpenReposter}>
-              <FontAwesomeIcon
-                icon="retweet"
-                style={{
-                  marginRight: 4,
-                  color: pal.colors.textLight,
-                  minWidth: 16,
-                }}
```

Comment:
These icons have a little bit of padding on them already, so decreasing the marginRight from 4 to 2